use reqwest::Error;

extern crate reqwest;
#[warn(unused_must_use)]
#[tokio::main]
async fn main() -> Result<(), Error> {
    // initialize::main().await;
    // verify::main().await;
    // charge::main().await;
    Ok(())
}



